tests/integration-tests/tests/schedulers/test_custom_munge_key/test_custom_munge_key/pcluster.config.yaml

```diff
@@ -0,0 +1,32 @@
+Image:
+  Os: {{ os }}
+LoginNodes:
+  Pools:
+    - Name: pool
+      InstanceType: {{ instance }}
+      Count: 1
+      Networking:
+        SubnetIds:
+          - {{ public_subnet_id }}
+      GracetimePeriod: 3
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+  Scheduler: slurm
+  SlurmSettings:
+    MungeKeySecretArn: {{ custom_munge_key_arn }}
+  SlurmQueues:
+    - Name: queue
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+      ComputeResources:
+        - Name: compute1
+          Instances:
+            - InstanceType: t2.medium
+          MinCount: 1
```

Comment:
If you are using 2 nodes in the test jobs above, then please just put 2 as min count (or set 0). Setting 1 is the worst possible choice here because you need to wait for one node to be provisioned at cluster creation, and one node at job submission, maximizing the duration of the test.

Reply:
Thanks for your reminding, great point! Done!